packages/base/src/3dview/mainview.tsx

```diff
@@ -829,7 +834,8 @@ export class MainView extends React.Component<IProps, IStates> {
             clientColor.g / 255,
             clientColor.b / 255
           )
-        : 'black'
+        : 'black',
+      wireframe: this.state.wireframe
```

Comment:
I would not apply the wireframe on the 3D mouse pointers

```suggestion
        : 'black'
```